lib/factory_girl/strategy/stub.rb

```diff
@@ -75,6 +75,28 @@ def created_at
             end
           end
         end
+
+        responds_to_updated_at = result_instance.respond_to?(:updated_at)
+        updated_at_missing_default = responds_to_updated_at && !result_instance.updated_at
+        result_instance_missing_updated_at = !responds_to_updated_at
+
+        if updated_at_missing_default || result_instance_missing_updated_at
+          result_instance.instance_eval do
+            def updated_at
+              @updated_at ||= Time.now.in_time_zone
```

Comment:
Thoughts on using `Time.current`, which is Rails' standard that takes timezones into account?